docs/rules/html-forbid-elements.md

```diff
@@ -0,0 +1,49 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/html-forbid-elements
+description: prevent use of forbidden html elements
+since: v8.6.0
+---
+
+# vue/html-forbid-elements
+
+> prevent use of forbidden html elements
+
+## :book: Rule Details
+
+This rule aims to disallow html elements that have been forbidden.
+
+<eslint-code-block fix :rules="{'vue/html-forbid-elements': ['error', { forbid: ['div', 'button'] } ] }">
```

Comment:
This rule has no effect without the option, so write it in the following order:

- `## :book: Rule Details`
- description of rule 
- `## :wrench: Options`
- description of options
- option example
- source code example